subsystem/src/main/java/org/wildfly/extension/elytron/tls/subsystem/_private/ElytronTLSLogger.java

```diff
@@ -0,0 +1,400 @@
+    void activatingSubsystem();
+
+    @Message(id = 2, value = "Deployment %s requires use of the '%s' capability but it is not currently registered")
+    DeploymentUnitProcessingException deploymentRequiresCapability(String deploymentName, String capabilityName);
+
+    @Message(id = 3, value = "The operation did not contain an address with a value for '%s'.")
+    IllegalArgumentException operationAddressMissingKey(final String key);
+
+    /**
+     * A {@link StartException} if it is not possible to initialise the {@link Service}.
+     *
+     * @param cause the cause of the failure.
+     * @return The {@link StartException} for the error.
+     */
+    @Message(id = 4, value = "Unable to start the service.")
+    StartException unableToStartService(@Cause Exception cause);
+
+    /**
+     * An {@link OperationFailedException} if it is not possible to access the {@link KeyStore} at RUNTIME.
+     *
+     * @param cause the underlying cause of the failure
+     * @return The {@link OperationFailedException} for the error.
+     */
+    @Message(id = 5, value = "Unable to access KeyStore to complete the requested operation.")
+    OperationFailedException unableToAccessKeyStore(@Cause Exception cause);
+
+    /**
+     * An {@link OperationFailedException} where an operation can not proceed as it's required service is not UP.
+     *
+     * @param serviceName the name of the service that is required.
+     * @param state the actual state of the service.
+     * @return The {@link OperationFailedException} for the error.
+     */
+    @Message(id = 7, value = "The required service '%s' is not UP, it is currently '%s'.")
+    OperationFailedException requiredServiceNotUp(ServiceName serviceName, ServiceController.State state);
+
+    /**
+     * An {@link OperationFailedException} where the name of the operation does not match the expected names.
+     *
+     * @param actualName the operation name contained within the request.
+     * @param expectedNames the expected operation names.
+     * @return The {@link OperationFailedException} for the error.
+     */
+    @Message(id = 8, value = "Invalid operation name '%s', expected one of '%s'")
+    OperationFailedException invalidOperationName(String actualName, String... expectedNames);
+
+    /**
+     * An {@link RuntimeException} where an operation can not be completed.
+     *
+     * @param cause the underlying cause of the failure.
+     * @return The {@link RuntimeException} for the error.
+     */
+    @Message(id = 9, value = "Unable to complete operation. '%s'")
+    RuntimeException unableToCompleteOperation(@Cause Throwable cause, String causeMessage);
+
+    /**
+     * An {@link OperationFailedException} where this an attempt to save a KeyStore without a File defined.
+     *
+     * @return The {@link OperationFailedException} for the error.
+     */
+    @Message(id = 10, value = "Unable to save KeyStore - KeyStore file '%s' does not exist.")
```

Comment:
I think we can update the ids in this case to make sure the numbers are all consecutive. Looks like there are some ids missing and the numbers jump since this was copied from the original subsystem. But since this is a new subsystem, we can just number normally.

Reply:
Any messages that aren't being used can also be removed.

Reply:
Sounds good, I mostly kept them as-is when I was originally copying since I wasn't sure if they were coordinated or not. I'll also reorganize them to make more sense.

Will leave this until credential store defintions, etc. are removed.